### PR TITLE
Prepare v0.13.0

### PR DESCRIPTION
**Pull request by seanmonstar**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,85 +1,133 @@
...
 
-#### Breaking Changes
+  `hyper::rt::run` usage should be replaced with `#[tokio::main]` or
+  managing a `tokio::runtime::Runtime` manually.
 
-* Using a `Body` as a `Stream`, and constructing one via
-  `Body::wrap_stream`, require enabling the `unstable-stream` feature.
+ ([6ae5889f](https://github.com/hyperium/hyper/commit/6ae5889f8378b6454d4dc620f33bd1678d0e00e4))
+* The `Resolve` trait is gone. All custom resolvers should
+  implement `tower::Service` instead.
```
Comment (LucioFranco) [inline CHANGELOG.md]:
```suggestion
  implement `tower_service::Service` instead.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,85 +1,133 @@
...
 
- ([511ea388](https://github.com/hyperium/hyper/commit/511ea3889b5cceccb3a42aa72465fe38adef71a4))
+  The error type of `HttpConnector` has been changed away from
+  `std::io::Error`.
+
+ ([9d9233ce](https://github.com/hyperium/hyper/commit/9d9233ce7ceddb0fa6f5e725b0a781929add3c58))
+* Any manual implementations of `Connect` must instead
+  implement `tower::Service<Uri>`.
```
Comment (LucioFranco) [inline CHANGELOG.md]:
```suggestion
  implement `tower_service::Service<Uri>`.
```
I think its better to push people to use the loweset possible crate for that.

**Comment by leo-lb** [comment]
Can't wait :-) -- Awesome work you're doing @seanmonstar

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,85 +1,133 @@
...
+  * update Tokio, bytes, http, h2, and http-body ([cb3f39c2](https://github.com/hyperium/hyper/commit/cb3f39c2dc6340060f6b17f354f04c872a947574))
+* **rt:** introduce `rt::Executor` trait ([6ae5889f](https://github.com/hyperium/hyper/commit/6ae5889f8378b6454d4dc620f33bd1678d0e00e4), closes [#1944](https://github.com/hyperium/hyper/issues/1944))
+* **server:**
+  * introduce `Accept` trait ([b3e55062](https://github.com/hyperium/hyper/commit/b3e5506261c33dcaca39a126e891a0b9d5df5eea))
+  * give `Server::local_addr` a more general type ([3cc93e79](https://github.com/hyperium/hyper/commit/3cc93e796aad59b3996fc26b8839a783e0307925))
+  * change `http1_half_close` option default to disabled ([7e31fd88](https://github.com/hyperium/hyper/commit/7e31fd88a86ac032d05670ba4e293e3e5fcccbaf))
+* **service:**
+    * use tower_service::Service for hyper::service ([ec520d56](https://github.com/hyperium/hyper/commit/ec520d5602d819fd92f497cc230df436c1a39eb0))
+  * rename `Service to `HttpService`, re-export `tower::Service` ([4f274399](https://github.com/hyperium/hyper/commit/4f2743991c227836c3886778512afe1297df3e5b), closes [#1959](https://github.com/hyperium/hyper/issues/1959))
```
Comment (kud1ing) [inline CHANGELOG.md]: a backquote is missing here

**Pull request merged**